managed/services/management/checks_test.go

```diff
@@ -460,3 +461,48 @@ func TestUpdateSecurityChecks(t *testing.T) {
 		assert.Equal(t, &managementpb.ChangeSecurityChecksResponse{}, resp)
 	})
 }
+
+func TestCreateComment(t *testing.T) {
+	t.Parallel()
+
+	l := logrus.WithField("component", "tests")
+
+	testCases := []struct {
```

Comment:
What is the expected behavior if there are no technologies supported?

Reply:
Every advisor has at least one check, so at least one technology is supported.